src/org/traccar/geocoder/JsonGeocoder.java

```diff
@@ -58,14 +64,14 @@ public void getAddress(
             }
         }
 
-        Context.getAsyncHttpClient().prepareGet(String.format(url, latitude, longitude))
+        Context.getAsyncHttpClient().prepareGet(String.format(Locale.US, url, latitude, longitude))
```

Comment:
I believe it is a bug. Because if default locale is not standard (like Russian) it formats numbers with comma instead of point and it always fail.

Reply:
We actually set locale in `Main` class. I guess it's not set for unit tests.